Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -170,6 +170,7 @@ namespace AzToolsFramework
         m_handler = nullptr;
         m_isMultiSizeContainer = false;
         m_isFixedSizeOrSmartPtrContainer = false;
+        m_custom = false;
```

Comment:
This looks unused

Reply:
the flag is set by TransformRowWidget.cpp and it disables a part of the PropertyRowWidget. I have to reset the flag to restore PropertyRowWidget when it goes back to the pool.

Reply:
OK sounds good!